File: Algorithm/01.Recursion/02.PrintMirrorTriangle/NegativeNumberException.java
public class NegativeNumberException extends Throwable {
   private static final String ANSI_RED = "\u001B[31m";
   private static final String RESET = "\u001B[0m";

   public void what() {
       System.out.println(ANSI_RED + "Cannot use negative number!" + RESET);
   }
}

File: TechnologiesFundamentals/ProgrammingFundamentals-Extended/DataTypesNumeralTypesAndTypeConversion-Lab/01.Factorial/01.Factorial/NegativeNumberException.cs
using System;
using System.Runtime.Serialization;

namespace _01.Factorial
{
    [Serializable]
    internal class NegativeNumberException : Exception
    {
        public NegativeNumberException()
        {
            Console.WriteLine("Cannot use negative number!");
        }

        public NegativeNumberException(string message) : base(message)
        {
        }

        public NegativeNumberException(string message, Exception innerException) : base(message, innerException)
        {
        }

        protected NegativeNumberException(SerializationInfo info, StreamingContext context) : base(info, context)
        {
        }
    }
}